### ISSUE TITLE
The music itself is skipped

### ISSUE THREAD

**Issue by MrPrezAtambaev** [KRTirtho/spotube#1876]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

when I play music, it skips by itself.

https://github.com/user-attachments/assets/0971db9a-3f99-48e3-9db0-766e4cbb22ff

### Expected Behavior

It should play all the music

### Steps to reproduce

I opened app 

play music

and I noticed this bug

### Operating System

Mac OS

### Spotube version

3.8.0

### Installation source

Website (spotube.krtirtho.dev)

### Self grab

- [ ] I'm ready to work on this issue!

**Comment by GrampaWildWilly** [comment]
I have noticed this on occasion.  It is very rare.  I believe this has something to do with copyrights.  There were tracks in my playlist on Spotify itself that were playable on the day I added them to the playlist.  Those tracks continue to be in my playlist.  But there are whole albums that simply won't play today even on Spotify itself.  I have always assumed this failure to play would transfer to Spotube.  Could this be the issue?

**Comment by MrPrezAtambaev** [comment]
when it is skipped, this error appears in the log


<img width="1062" alt="Снимок экрана 2024-09-16 в 12 17 07" src="https://github.com/user-attachments/assets/ff7a2295-286a-4721-b7c8-f972b582f0c3">

**Comment by GrampaWildWilly** [comment]
When you get on Spotify itself, is L'Étoile d'Afrique playable?

Understand, I'm just another user, like you.  Your problem description just caught my eye.

**Comment by MrPrezAtambaev** [comment]
yes, it plays on spotify

**Comment by GrampaWildWilly** [comment]
Right.  While you were posting that, I was checking it out myself.  It plays for me, too.  Do you get the same error consistently in Spotube?  If you try again now, do you still get the same error in Spotube?

**Comment by MrPrezAtambaev** [comment]
some music plays after a while, and some do not play at all

**Comment by GrampaWildWilly** [comment]
Oh great.  An intermittent problem.  I'm glad I'm not developing this software.  😄

**Comment by MrPrezAtambaev** [comment]
for example, this music does not play "malevolencia immortal" at all, even if I change the audio source
